idaes/models_extra/power_generation/costing/tests/test_CCS_capcost.py

```diff
@@ -43,15 +47,14 @@
 
 
 @pytest.mark.component
-def test_remove_existing_dictionaries():
-    # Remove the dictionaries so they may be regenerated during testing
-    dict_path = os.path.join(
-        os.path.join(directory, ".."), "generic_ccs_costing_data.json"
-    )
-    assert os.path.exists(dict_path)
+def test_regenerate_ccs_dictionaries():
+    # Switch to a temporary directory to test regenerating the dictionaries
+    _temp_context = TempfileManager.new_context()
+    wrkdir = _temp_context.create_tempdir()
+
+    load_generic_ccs_costing_dictionary(path=wrkdir)
 
```

Comment:
There needs to be some check of what is generated here. You should at least check that the expected files were generated, and maybe even check that they look right.

Also, if the `load_generic_ccs_costing_dictionary` method returns something, you should check what is returned.